src/icons/ArcText.tsx

```diff
@@ -22,8 +22,9 @@ interface ArcTextProps {
 
 export default function ({ text, mobile }: ArcTextProps) {
   const { xxs, md } = useBreakpoints()
-  const radius = 85
+  const radius = mobile ? 85 : 110
 
+  console.log('mobile: ', mobile)
```

Comment:
Don't push unnecessary `console.log` statements

Reply:
Removed it